docs/source/conf.py

```diff
@@ -26,8 +26,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'btrdb'
-copyright = '2019, Michael P. Andersen'
-author = 'Michael P. Andersen'
+copyright = '2021, PingThingsIO'
```

Comment:
pingthings here should be referred to as `Ping Things, Inc.`.  Note the space.

Reply:
sounds good. Should I change it to that everywhere or just the copyright line?